### DISCUSSION TITLE
Add title to the connection

### DISCUSSION THREAD

**Discussion by carolinaspb** [joyceworks/flowchart-react#30] [Q&A] [answered]
Hi! Can you please add a title to the connection? Something like that 
![image](https://user-images.githubusercontent.com/108330421/209821774-d8c4c307-fd46-4fe3-b40e-9b46bae9f8bc.png)

**Comment by iamppz** [comment]
OK, I’ll do it later.

获取 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: carolinaspb ***@***.***>
发送时间: Wednesday, December 28, 2022 9:47:21 PM
收件人: joyceworks/flowchart-react ***@***.***>
抄送: Subscribed ***@***.***>
主题: [joyceworks/flowchart-react] Add title to the connection (Discussion #30)


Hi! Can you please add a title to the connection? Something like that
[image]<https://user-images.githubusercontent.com/108330421/209821774-d8c4c307-fd46-4fe3-b40e-9b46bae9f8bc.png>

―

**Comment by iamppz** [comment, marked as answer]
Please upgrade flowchart-react to 3.10.1 and add `title` to the connections:
```
const connections = [
  {
    source: {id: 1, position: "right"},
    destination: {id: 4, position: "left"},
    type: "success",
    title: "1"
  },
  ...
]
```
![image](https://user-images.githubusercontent.com/5696485/210164285-f91f4058-8c27-44ad-991a-40626fd534fb.png)

**Reply by carolinaspb** [reply]
Thanks a lot!
